### PR TITLE
Fixed bug that wasn't enabling to click behind notification div

### PR DESCRIPTION
**Pull request by yuriburk**
### What does it do?

Fix a bug that when a notification is showing on screen, you can't click in anything that is behind the invisible fixed div.

### Why is it needed?

To click on anything that is behing the invisible div, like the first collection

### How to test it?

Run the strapi with a notification on

### Video

https://user-images.githubusercontent.com/11825151/109582139-64f36a80-7adc-11eb-8b81-6f8acfe8d25f.mp4

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Notifications/NotificationsContainer/Wrapper.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const Wrapper = styled(TransitionGroup)`
   width: 300px;
   margin: 0 auto;
   overflow-y: hidden;
+  pointer-events: none;
```
Comment (soupette) [inline packages/strapi-admin/admin/src/components/Notifications/NotificationsContainer/Wrapper.js:14]: Can you also add the property to the `strapi-admin/admin/src/containers/NewNotification/Wrapper.js` so the bug to fix the issue with the new API also?

**Review by soupette — CHANGES_REQUESTED** [review]
Thank you for your contribution, can you apply the requested changes so we can merge your PR?

**Review by HichamELBSI — APPROVED** [review]
LGTM

File: packages/strapi-admin/admin/src/components/Notifications/NotificationsContainer/Wrapper.js (hunk 1 above)
Comment (yuriburk) [reply packages/strapi-admin/admin/src/components/Notifications/NotificationsContainer/Wrapper.js:14]: Hi! Just done the change.

**Comment by yuriburk** [comment]
Requested change done @soupette @HichamELBSI , thank you for the review

**Review by soupette — APPROVED** [review]
LGTM!

Thank you for your contribution!

**Pull request merged**